package.json

```diff
@@ -55,6 +55,7 @@
 		"commitplease": "2.3.0",
 		"grunt": "0.4.5",
 		"grunt-bowercopy": "1.2.4",
+		"grunt-cli": "0.1.13",
```

Comment:
How does it break it? You can still use the global `grunt-cli` if you wish so. OTH:

1. In development all the dependencies are fixed so the possibility of a breakage is decreased. npm scripts can use the `grunt` binary without any global installation.
2. People that don't like global packages may not use them here. Without including `grunt-cli` it's simply not possible as the `grunt` package doesn't contain any binary to run.

Most of the projects I contribute to include `grunt-cli` in `devDependencies` for these reasons. I often don't even have `grunt-cli` installed globally until I encounter a project that doesn't include this.

It seems to me that including it as a dependency doesn't break the usage for people that don't want to rely on it but not including it breaks the usage for others.

Even the `grunt-cli` [README](https://github.com/gruntjs/grunt-cli#installing-grunt-cli-locally) mentions this pattern so I don't understand your comment.